### DISCUSSION TITLE
Unified "Download" button

### DISCUSSION THREAD

**Discussion by BrownTurtleBear** [modrinth/.github#48] [Feature requests]
Right now we can update a texturepack and say yes it can be played on what ever versions. However Because each major update/some minor updates have the texturepack number change the pack would come out red on the texturepack list. So there should be a way for us to upload All the different versions for Minecraft on the one version for your  texturepacks. For the people that are downloading the packs they would have a dropdown button with all the versions they can download the pack for the click it, then download. You can change this to a text box for them to manually search for the version.

**Comment by triphora** [comment]
Duplicate of #53

**Reply by BrownTurtleBear** [reply]
> Duplicate of #53
I'm sorry but we are talking about way two different things. This is not a duplicate.

**Reply by Gaming32** [reply]
Yeah, this is not a duplicate at all.
